rasa/core/channels/slack.py

```diff
@@ -182,7 +184,10 @@ def __init__(
             slack_signing_secret: Slack creates a unique string for your app and
                 shares it with you. This allows us to verify requests from Slack
                 with confidence by verifying signatures using your signing secret.
-
+            conversation_id_config: conversation id config for slack conversations.
```

Comment:
Config has the connotation of a more general configuration, sometimes a file or a larger amount of options. Would another name such as `conversation_level` or `conversation_granularity` (with values being `sender`, `channel`, `thread`) make more sense here?

Also it makes sense to set and talk about the default here in my opinion.